packages/vscode/package.json

```diff
@@ -17,6 +17,7 @@
     "checkpoint-client": "1.1.11",
     "crypto": "1.0.1",
     "chokidar": "3.4.2",
+    "fsevents": "2.1.3",
```

Comment:
I hope that this won't bother other OSes

Reply:
it is bothering Windows and Ubuntu unfortunately on `npm install` 🙈